### ISSUE TITLE
Tree view shows spinner and never loads

### ISSUE THREAD

**Issue by dmcintyr** [Nitrate/Nitrate#290]
I installed the Vagrant image on MacOS and launched Nitrate. Most functions work, but the tree view shows a spinner and never loads. The debug console shows that jquery.min.js is error 404. I added a symlink back to js/lib/jquery-1.7.2.min.js, but got new errors, as follows:
```
jquery-1.7.2.min.js:4 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
send @ jquery-1.7.2.min.js:4
ajax @ jquery-1.7.2.min.js:4
filter @ testplan_actions.js:31
init @ testplan_actions.js:58
loadPlansTreeView @ testplan_actions.js:719
(anonymous) @ testplan_actions.js:905
dispatch @ jquery-1.7.2.min.js:3
i @ jquery-1.7.2.min.js:3
fireEvent @ tcms_actions.js:774
initTabs @ testplan_actions.js:744
on_load @ testplan_actions.js:992
dispatch @ jquery-1.7.2.min.js:3
i @ jquery-1.7.2.min.js:3
jquery-1.7.2.min.js:4 GET http://10.34.64.99:8087/plans/?pk=1&t=ajax 500 (Internal Server Error)
send @ jquery-1.7.2.min.js:4
ajax @ jquery-1.7.2.min.js:4
filter @ testplan_actions.js:31
init @ testplan_actions.js:58
loadPlansTreeView @ testplan_actions.js:719
(anonymous) @ testplan_actions.js:905
dispatch @ jquery-1.7.2.min.js:3
i @ jquery-1.7.2.min.js:3
fireEvent @ tcms_actions.js:774
initTabs @ testplan_actions.js:744
on_load @ testplan_actions.js:992
dispatch @ jquery-1.7.2.min.js:3
i @ jquery-1.7.2.min.js:3
jquery-1.7.2.min.js:2 Uncaught SyntaxError: Unexpected token T in JSON at position 0
    at JSON.parse (<anonymous>)
    at Function.parseJSON (jquery-1.7.2.min.js:2)
    at json_failure (tcms_actions.js:223)
    at Object.error (testplan_actions.js:40)
    at o (jquery-1.7.2.min.js:2)
    at Object.fireWith [as rejectWith] (jquery-1.7.2.min.js:2)
    at w (jquery-1.7.2.min.js:4)
    at d (jquery-1.7.2.min.js:4)
    at Object.send (jquery-1.7.2.min.js:4)
    at Function.ajax (jquery-1.7.2.min.js:4)
parseJSON @ jquery-1.7.2.min.js:2
json_failure @ tcms_actions.js:223
error @ testplan_actions.js:40
o @ jquery-1.7.2.min.js:2
fireWith @ jquery-1.7.2.min.js:2
w @ jquery-1.7.2.min.js:4
d @ jquery-1.7.2.min.js:4
send @ jquery-1.7.2.min.js:4
ajax @ jquery-1.7.2.min.js:4
filter @ testplan_actions.js:31
init @ testplan_actions.js:58
loadPlansTreeView @ testplan_actions.js:719
(anonymous) @ testplan_actions.js:905
dispatch @ jquery-1.7.2.min.js:3
i @ jquery-1.7.2.min.js:3
fireEvent @ tcms_actions.js:774
initTabs @ testplan_actions.js:744
on_load @ testplan_actions.js:992
dispatch @ jquery-1.7.2.min.js:3
i @ jquery-1.7.2.min.js:3

```

**Comment by tkdchen** [comment]
Thanks for reporting. It's really a bug.

**Comment by tkdchen** [comment]
Fixed in #301
